sheepdog/utils/manifest.py

```diff
@@ -31,7 +31,7 @@
                         "type": "string"
                     },
                     "file_size": {
-                        "type": "integer"
+                        "type": "number"
```

Comment:
I need to verify in ticket, but we don't just want to be patching this. file_size shouldn't be a number. The issue is that in some imports, it comes in as a number, and this is just a hack to get around the problem.